### DISCUSSION TITLE
Antler enabled field value on Blade template

### DISCUSSION THREAD

**Discussion by saqueib** [statamic/cms#8696] [Q&A] [answered]
I have a field which needs to pass title as variable which is antlers enabled. I want to use the parsed value in a blade template, how can I achieve it.

```md
content: 'Hi! Welcome to {{ $title }}'
```

I am querying this entry using

```php
 $path = $request->path();
 $entry = Entry::query()
            ->where('uri', "/" . $path)
            ->first();
```

and in blade template

```html
<h3> {{ $entry->content }} </h3>
```

but Its giving me raw value without rendering value for variable **title**

Output
```html
<h3>Hi! Welcome to {{ $title }}</h3>
```

**Comment by jackmcdade** [comment]
Hmm, we have a "parse Antlers" setting on all fields to allow variables to be used in content, but I don't think that feature exists for Blade. Can you enable that setting in your blueprint and then try an Antlers template (even a partial will work with a regular @include()` and see if it works as expected there?

**Reply by saqueib** [reply]
Thanks @jackmcdade, tried include antlers template but its not working. its removing the antler syntax but not printing title value.

Its now printing `<h3>Hi! Welcome to </h3>`

**Blueprint**
```yaml
...
type: markdown
icon: markdown
antlers: true
```

**web route**
```php
return view('test', ['entry' => $entry]);
```

**test.blade.php** blade view
```blade
<h1>Testing antler parse</h1>
@include('block', [...$entry->toArray(), ['title' => 'Testing title']]))
```

**Antlers partials**
block.antlers.html
```antlers
<h1>Hello</h1>
{{ $content }}
```

Is there any other way like Facade to render antlers value from string, like `Antlers::render('Hi! Welcome to {{ $title }}', ['title' => 'Jack Stats'])`??

**Reply by ryanmitchell** [reply]
There is indeed: \Statamic\Facades\Antlers::parse('string', $data);

**Reply by saqueib** [reply]
@ryanmitchell I think that can work, is there anyway I can check for a field is enabled for antlers in blade template so I can do something like this

```blade
@if($entry->content->antlers)
\Statamic\Facades\Antlers::parse($entry->content, ['title' => 'Jack Stats'])
@endif
```

I can see this in dump 
![antler config](https://github.com/statamic/cms/assets/6937224/fb6da510-9a1e-4f59-9211-543849324c24)

**Reply by ryanmitchell** [reply, marked as answer]
You should be able to do something like `$entry->blueprint()->fields()->get($your_field_handle)->config('antlers')`

**Comment by jackmcdade** [comment]
Looks like this is a known issue: https://github.com/statamic/cms/issues/8430
